nathanielmillard/rancid-tomatillos
# Nd basic components structure

#### What’s this PR do?

- This PR builds out the basic structure for the App, featuring basic styling.
- Installs node-sass to allow us to use scss within the project.
- Sets a basic state for a user to test conditional logic for the navbar.
- Created MovieMain Component that is a class component.
- MovieMain state fetches the data from the api `https://rancid-tomatillos.herokuapp.com/api/v2/movies` to display movies to the movie-directory section of the app.
- Created MovieTile Component that is a stateless component to display each individual instance of a movie pulled from the movie API.
- Converts App from a functional component to a class component with state.

#### Where should the reviewer start?
- Git pull this branch.
- Will have to run `npm install` to install the node-sass library.

#### How should this be manually tested?
- After running `npm install`, `npm start` to pull up server.
- Navigate to localhost:3000.
- Should be able to see a red bar on the left side of the window representing navbar with basic data inside regarding a user.
- Should be able to see each individual movie poster that is pulled from the API (should be around 40 of them) displaying the movie poster and rating below the poster.

#### Any background context you want to provide?
- N/A

#### What are the relevant tickets?
- closes #8 
- closes #7 
- closes #6

#### Screenshots (if appropriate):
- n/a

#### Questions:
- n/a
